flight/test_files/README.md

```diff
@@ -0,0 +1,8 @@
+**Test Files Subdirectory**
+
+Contains files and coordinates for running test flights at both the golf course and David Hoffman's land.
+
+Both files have basic drone functions for creating connection with drone,
+logging takeoff status, flying to each given waypoint, and returning home.
+
+Altitude is set to 100 feet for both.
```

Comment:
This isn't entirely true, the testing file have multiple altitudes for different waypoints.